.gitignore

```diff
@@ -1,3 +1,4 @@
+*~
```

Comment:
Is this for temp files created by your editor?  If so, I think it's better to have this in your global system gitignore.

```sh
git config --global core.excludesfile ~/.gitignore_global
```

I only mention because I use a variety of editors and if I added gitignore entries on every project I worked on for the cruft they all leave, it would be quite significant.  Sorry if I assume incorrectly.